src/static/actions/cards.js

```diff
@@ -9,6 +9,15 @@ import { USER_CARDS_FETCH_REQUEST,
     CARDS_FETCH_REQUEST,
     CARDS_RECEIVE,
     CARDS_FETCH_FAILURE,
+    CARD_FETCH_REQUEST,
+    CARD_RECEIVE,
+    CARD_FETCH_FAILURE,
+    CARD_DELETE_FAILURE,
+    CARD_DELETE_REQUEST,
+    CARD_DELETE_SUCCESS,
+    CARD_EDIT_FAILURE,
+    CARD_EDIT_REQUEST,
+    CARD_EDIT_SUCCESS,
     CARD_ADD_FAILURE,
     CARD_ADD_REQUEST,
     CARD_ADD_SUCCESS } from '../constants';
```

Comment:
Maybe use a:
import * as constants from '../constants';
and use it as constants.CARD_EDIT_FAILURE for example.
It's up to you.

Reply:
Done in the next branch